src/styles/card.scss

```diff
@@ -0,0 +1,106 @@
+$text-color: #616070;
+$button-text-color: #00acdc;
+
+.card {
+  display: flex;
+  flex-direction: column;
+  width: 200px;
+  box-sizing: border-box;
+  border: solid 1px #f3f3f3;
+  border-radius: 5px;
+  padding-inline: 16px;
+  transition: all 300ms linear;
+
+  &:hover{
+    transform: scale(1.2);
+  }
```

Comment:
The hover effect on the card is too aggressive. A scaling of 1.2 is too much and can lead to a confusing user experience. Consider reducing it to a more subtle effect, like 1.05 or 1.1.